Tests/base/NSPersonNameComponentsFormatter/GNUmakefile

```diff
@@ -0,0 +1,29 @@
+# __GENERATED__ makefile marker
```

Comment:
I think this is a generated file, it should not be on git. The same seems to be true about the next file as well.

Reply:
Removed